dts/arm/st/l1/stm32l152.dtsi

```diff
@@ -1,11 +1,7 @@
 /*
- * Copyright (c) 2019 Antony Pavlov <antonynpavlov@gmail.com>
+ * Copyright (c) 2020 Abel Sensors
  *
  * SPDX-License-Identifier: Apache-2.0
  */
 
-/*
- * The STM32L151 and STM32L152 differ in that
- * the STM32L152 features an LCD controller.
- */
-#include <st/l1/stm32l151.dtsi>
```

Comment:
Can you explain this change ?
I don't think it is needed.

Reply:
It is not needed for it to work, but this is much more in line with the abstraction 'levels' when looking at the other SoC models. If basing it on 151 is preferred, I can change it back, but I would say this is preferred since it might help overcome abstraction issues later on (if things need to be implemented that are specifically for 152, and not 151 etc)